internal/driver/driver.go

```diff
@@ -225,6 +225,14 @@ func reportOptions(p *profile.Profile, vars variables) (*report.Options, error)
 		return nil, fmt.Errorf("zero divisor specified")
 	}
 
+	var filters []string
+	for _, k := range []string{"focus", "ignore", "hide", "show", "tagfocus", "tagignore", "tagshow", "taghide"} {
+		v := vars[k].value
+		if v != "" {
+			filters = append(filters, "   "+k+"="+v)
```

Comment:
I think adding the leading spaces should be pushed down to where the legend formatting is done. The caller here doesn't know that the entries are going to be displayed one per line, they could as well be semicolon separated or something, where the leading spaces are unwanted.